projects/Mallard/ios/Mallard/Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>Guardian Daily</string>
+	<string>$(PRODUCT_NAME)</string>
```

Comment:
I wish this was the only change required for iOS (rest of the code should just read from it, like android)

Reply:
This is what XCode did automatically. I prefer to only edit these files manually when I absolutely have to.